src/shared/chrome.js

```diff
@@ -0,0 +1,11 @@
+export function promisifiedChromeStorageSyncGet() {
+  return new Promise((resolve, reject) => {
+    chrome.storage.sync.get(data => {
+      if (chrome.runtime.lastError) {
+        reject(chrome.runtime.lastError);
+      }
+
+      resolve(data);
+    });
+  });
+}
```

Comment:
Maybe we could write a whole high-level Promise-based API on top of the `chrome` API so we don't have to deal with stubbing as much in tests, doesn't have to be in the PR though

Reply:
Sure, that seems reasonable

Reply:
I usually use TypeScript now but I can give this a shit

Reply:
If we do that, it might be nice for one of us to just create a separate package that this project consumes - I’m sure others would use it :)

Reply:
I originally suggested [webextension-polyfill](https://github.com/mozilla/webextension-polyfill) for Firefox support, but it could be useful for this as it brings Firefox's Promise-based extension API to Chrome

Reply:
Oh yeah, that sounds like a good idea!

Reply:
Started in #99